get_census_data.py

```diff
@@ -28,6 +28,9 @@ def getAllCounties():
   # For every state, get all counties
   for state in states:
     print("getting: ", state)
+    if state == "Alaska":
```

Comment:
there's an entry for DC in `census_data.csv` — you could filter it out with alaska here